src/feed/SubscriptionManagerInterface.hpp

```diff
@@ -234,7 +234,6 @@ class SubscriptionManagerInterface {
 
     /**
      * @brief Get the number of subscribers.
-     *
```

Comment:
Probably accidental change

Reply:
Actually it wasn't apparently. But i'll bring it back and change the other doxy comment that did not have the space line instead :rage4: